sample_code/ep354/README.md

```diff
@@ -0,0 +1,16 @@
+# [git without `cd` (intermediate)](https://youtu.be/fQ3-Y99Fzro)
+
+Today I show a very quick trick for running git commands in other directories without requiring `cd`!
+
+## Interactive examples
+
+### Bash
+
+```bash
+git init pre-commit
+git init all-repos
```

Comment:
Added those two for a functional example. Do you prefer them like empty repos or you prefer the `git clone ...` equivalent commands, the latter will be slower and will require network :)

Reply:
this is fine